build.sbt

```diff
@@ -132,14 +153,25 @@ lazy val core = crossProject(JSPlatform, JVMPlatform)
   .jsSettings(
     libraryDependencies ++= Seq(
       "org.scala-js" %%% "scalajs-dom" % "0.9.5"
+    ),
+    jsDependencies ++= Seq(
+      "org.webjars.npm" % "spark-md5" % "3.0.0" % "test" / "spark-md5.js" minified "spark-md5.min.js"
     )
   )
+  .jsSettings(browserTestSettings)
+  .jsSettings(testServerSettings(Test))
   .jvmSettings(
     libraryDependencies ++= Seq(
       akkaHttp % "test",
+      "ch.megard" %% "akka-http-cors" % "0.3.0" % "test",
       akkaStreams % "test",
       "org.scala-lang" % "scala-compiler" % scalaVersion.value % "test"
-    )
+    ),
+    mainClass in reStart := Some("com.softwaremill.sttp.testing.HttpServer"),
```

Comment:
the revolver configuration is for development as I understand, not strictly needed for tests? If so, a comment here would be useful as well

Reply:
Done